pom.xml

```diff
@@ -86,6 +77,11 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
```

Comment:
`mvn dependency:analyze` reports `org.apache.commons:commons-collections4:jar:4.3:compile` as an unused declared dependency.

Reply:
Interesting: not sure how to satisfy it. Some scope issue? Or a false positive?